packages/nc-gui/components/dashboard/TreeView/TableNode.vue

```diff
@@ -231,7 +231,10 @@ const isTableOpened = computed(() => {
           </div>
         </div>
       </div>
-      <NcTooltip class="nc-tbl-title nc-sidebar-node-title text-ellipsis w-full overflow-hidden select-none">
+      <NcTooltip
+        class="nc-tbl-title nc-sidebar-node-title text-ellipsis w-full overflow-hidden select-none"
+        :showOnTruncateOnly="true"
```

Comment:
In vue, u can just use `showOnTruncateOnly` instead of `:showOnTruncateOnly="true"`. 

Also use kebab syntax for prop name on template side. Vue will automatically convert the cases.

```
<NcTooltip
         class="nc-tbl-title nc-sidebar-node-title text-ellipsis w-full overflow-hidden select-none"
         show-on-truncate-only
```

Reply:
**FIXED IN BELOW COMMIT**

[fix : used kebab syntax for showOnTruncateOnly prop](https://github.com/nocodb/nocodb/pull/7163/commits/74da22eeeca3fa1e7af1addb12820bde0b3d2c0c)
[74da22e](https://github.com/nocodb/nocodb/pull/7163/commits/74da22eeeca3fa1e7af1addb12820bde0b3d2c0c)